src/main/java/io/seqera/tower/cli/commands/workspaces/ListCmd.java

```diff
@@ -38,15 +38,15 @@
     protected Response exec() throws ApiException, IOException {
         ListWorkspacesAndOrgResponse response = api().listWorkspacesUser(userId());
 
-        if (organizationName != null && response.getOrgsAndWorkspaces() != null) {
-            List<OrgAndWorkspaceDbDto> responseOrg = response.getOrgsAndWorkspaces()
-                    .stream()
-                    .filter(item -> Objects.equals(item.getOrgName(), organizationName))
-                    .collect(Collectors.toList());
-
-            return new WorkspaceList(userName(), responseOrg);
+        if (response.getOrgsAndWorkspaces() == null) {
+            return new WorkspaceList(userName(), Collections.emptyList());
         }
 
-        return new WorkspaceList(userName(), response.getOrgsAndWorkspaces());
+        List<OrgAndWorkspaceDbDto> responseOrg = response.getOrgsAndWorkspaces()
+                .stream()
+                .filter(v -> v.getWorkspaceId() != null && (organizationName == null || organizationName.equals(v.getOrgName()))
```

Comment:
What is purpose of the  `organizationName == null` condition?

Reply:
When `organizationName` is `null` we don't want to apply this filter `organizationName.equals(v.getOrgName())` but otherwise we want to apply it (this is why I'm using the OR). The new thing here is to filter out the ones that have a `null` `v.getWorkspaceId()`.

Reply:
I just can't imagine a scenario with a nullable organization name. Was wondering whether if it was redundant.

Reply:
Si que puede ser `null`. Cuando haces un `tw workspaces list` sin el `-o` que es opcional, te devuelve una lista con todos los workspaces en todas las orgs.